charts/ozone/templates/om/om-statefulset.yaml

```diff
@@ -31,6 +31,7 @@ metadata:
     app.kubernetes.io/component: om
 spec:
   replicas: {{ .Values.om.replicas }}
+  podManagementPolicy: Parallel
```

Comment:
As per [OM documentation](https://ozone.apache.org/docs/current/feature/om-ha.html)
>To convert a non-HA OM to be HA or to add new OM nodes to existing HA OM ring, new OM node(s) need to be bootstrapped.

Shouldn't there be an init container which calls `--bootstrap` for OM?

Reply:
Yes, I think you are right.

Reply:
At this point, I get stuck in the bootstrap init container. I believe this step is only necessary for new cluster nodes. Thus, the problem reoccurs when we scale (once again, the scaling issue).